streaming/src/main/scala/org/apache/spark/streaming/receiver/Receiver.scala

```diff
@@ -271,7 +271,7 @@ abstract class Receiver[T](val storageLevel: StorageLevel) extends Serializable
   }
 
   /** Get the attached executor. */
-  private def executor = {
+  private[streaming] def executor = {
```

Comment:
I didn't know about `PrivateTester`, I'll look into it. Other Spark modules (such as core) are using these access modifiers, so I thought it's common practice.